onadata/settings/base.py

```diff
@@ -37,10 +39,7 @@ def skip_suspicious_operations(record):
 ################################
 
 # Django `SECRET_KEY`
-try:
-    SECRET_KEY = os.environ['DJANGO_SECRET_KEY']
-except KeyError:
-    raise Exception('DJANGO_SECRET_KEY must be set in the environment.')
+SECRET_KEY = env("DJANGO_SECRET_KEY")
```

Comment:
Let's keep the single-quote ;-) 

For the reference: It's ok to get rid of the try/except KeyError since Django will refuse to start with a `ImproperlyConfigured` error.